src/ui/components/stepCard/StepCard.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement, useMemo } from 'react'
+import styled, { DefaultTheme, useTheme } from 'styled-components/native'
+
+import { getSpacing, Typo } from 'ui/theme'
+
+export enum StepCardType {
+  DONE = 'done',
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+interface StepCardProps {
+  title: string
+  icon: ReactElement
+  subtitle?: string
+  type?: StepCardType
+}
+
+export const StepCard: React.FC<StepCardProps> = ({
+  title,
+  subtitle,
+  icon,
+  type = StepCardType.ACTIVE,
+}) => {
```

Comment:
Pour compléter le commentaire de Cédric :

```suggestion
export function StepCard ({
  title,
  subtitle,
  icon,
  type = StepCardType.ACTIVE,
}: StepCardProps) => {
```